### ISSUE TITLE
The app doesn't build with node 17

### ISSUE THREAD

**Issue by PolpOnline** [gitify-app/gitify#530]
Apparently, some dependencies fail to build in node 17+
When I try to build the app with `yarn build` in node 17.1.0 the following error log is generated:

<details>
<summary>Click to expand</summary>

```
 λ yarn build
  yarn run v1.22.17
  $ webpack --config webpack.prod.js
  [webpack-cli] Compilation finished
  assets by status 552 KiB [cached] 1 asset
  orphan modules 1.04 MiB [orphan] 300 modules
  runtime modules 780 bytes 4 modules
  built modules 1.56 MiB [built]
    cacheable modules 1010 KiB 111 modules
    modules with errors 78 bytes [errors]
      ./node_modules/css-loader/dist/cjs.js!./node_modules/postcss- 
 loader/dist/cjs.js!./node_modules/tailwindcss/tailwind.css 39 bytes [not cacheable] [built] [code generated] 
  [1 error]
      ./node_modules/css-loader/dist/cjs.js!./node_modules/postcss- 
  loader/dist/cjs.js!./node_modules/nprogress/nprogress.css 39 bytes [not cacheable] [built] [code generated] 
  [1 error]
    ./src/index.tsx + 87 modules 590 KiB [not cacheable] [built] [code generated]
    external "electron" 42 bytes [built] [code generated]
  
  ERROR in ./node_modules/nprogress/nprogress.css (./node_modules/css- 
 loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./node_modules/nprogress/nprogress.css)
  Module build failed (from ./node_modules/css-loader/dist/cjs.js):
  Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in 
  /home/tommy/Documents/Git_projects/gitify/node_modules/postcss/package.json
      at new NodeError (node:internal/errors:371:5)
      at throwExportsNotFound (node:internal/modules/esm/resolve:429:9)
      at packageExportsResolve (node:internal/modules/esm/resolve:683:3)
      at resolveExports (node:internal/modules/cjs/loader:482:36)
      at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:999:19)
      at require (/home/tommy/Documents/Git_projects/gitify/node_modules/v8-compile-cache/v8-compile- 
  cache.js:159:20)
      at Object.<anonymous> (/home/tommy/Documents/Git_projects/gitify/node_modules/css- 
 loader/dist/index.js:12:39)
   @ ./node_modules/nprogress/nprogress.css 2:12-108 9:17-24 13:15-29
   @ ./src/index.tsx 4:0-33
  
  ERROR in ./node_modules/tailwindcss/tailwind.css (./node_modules/css- 
 loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./node_modules/tailwindcss/tailwind.css)
  Module build failed (from ./node_modules/css-loader/dist/cjs.js):
  Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in 
  /home/tommy/Documents/Git_projects/gitify/node_modules/postcss/package.json
      at new NodeError (node:internal/errors:371:5)
      at throwExportsNotFound (node:internal/modules/esm/resolve:429:9)
      at packageExportsResolve (node:internal/modules/esm/resolve:683:3)
      at resolveExports (node:internal/modules/cjs/loader:482:36)
      at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
      at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
      at Function.Module._load (node:internal/modules/cjs/loader:778:27)
      at Module.require (node:internal/modules/cjs/loader:999:19)
      at require (/home/tommy/Documents/Git_projects/gitify/node_modules/v8-compile-cache/v8-compile- 
  cache.js:159:20)
      at Object.<anonymous> (/home/tommy/Documents/Git_projects/gitify/node_modules/css- 
 loader/dist/index.js:12:39)
   @ ./node_modules/tailwindcss/tailwind.css 2:12-107 9:17-24 13:15-29
   @ ./src/index.tsx 3:0-34

  webpack 5.11.0 compiled with 2 errors in 43343 ms
  error Command failed with exit code 1.
  info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
</details>

**Comment by bmulholland** [comment]
Now upgraded to Node 18. @afonsojramos can we close this one out?

**Comment by afonsojramos** [comment]
Yep! Closed on #566.
